### ISSUE TITLE
Khard crashes wen trying to edit a contact

### ISSUE THREAD

**Issue by Hund** [scheibler/khard#100]
I'm running Arch Linux, Khard 0.11.3 and Vdirsyncer 0.13.1. 

When I edit a contact and save the file Khard crashes:

```
Traceback (most recent call last):
  File "/usr/bin/khard", line 11, in <module>
    load_entry_point('khard==0.11.3', 'console_scripts', 'khard')()
  File "/usr/lib/python3.5/site-packages/khard/khard.py", line 1675, in main
    args.open_editor)
  File "/usr/lib/python3.5/site-packages/khard/khard.py", line 967, in modify_subcommand
    modify_existing_contact(selected_vcard)
  File "/usr/lib/python3.5/site-packages/khard/khard.py", line 129, in modify_existing_contact
    new_contact.write_to_file(overwrite=True)
  File "/usr/lib/python3.5/site-packages/khard/carddav_object.py", line 1421, in write_to_file
    vcard_string = self.vcard.serialize()
  File "/usr/lib/python3.5/site-packages/vobject/base.py", line 255, in serialize
    return behavior.serialize(self, buf, lineLength, validate)
  File "/usr/lib/python3.5/site-packages/vobject/behavior.py", line 162, in serialize
    out = base.defaultSerialize(transformed, buf, lineLength)
  File "/usr/lib/python3.5/site-packages/vobject/base.py", line 993, in defaultSerialize
    child.serialize(outbuf, lineLength, validate=False)
  File "/usr/lib/python3.5/site-packages/vobject/base.py", line 255, in serialize
    return behavior.serialize(self, buf, lineLength, validate)
  File "/usr/lib/python3.5/site-packages/vobject/vcard.py", line 227, in serialize
    VCardTextBehavior.serialize(obj, buf, lineLength, validate)
  File "/usr/lib/python3.5/site-packages/vobject/behavior.py", line 162, in serialize
    out = base.defaultSerialize(transformed, buf, lineLength)
  File "/usr/lib/python3.5/site-packages/vobject/base.py", line 1001, in defaultSerialize
    obj.behavior.encode(obj)
  File "/usr/lib/python3.5/site-packages/vobject/vcard.py", line 152, in encode
    line.value = codecs.encode(line.value.encode(coding), "base64").decode("utf-8")
AttributeError: 'bytes' object has no attribute 'encode'
```

What could be wrong?

**Comment by lucc** [comment]
What is your vobject version? khard currently needs an older version, see the khard setup.py file.

**Comment by Hund** [comment]
It's version	0.9.3. 

I see, I'll install a older version then. :)

**Comment by scheibler** [comment]
Fixed in khard release 0.11.4
